### PR TITLE
Squashed commit for version 7.x-1.5-alpha1 of ucla_tab_search and ucl…

### PR DESCRIPTION
**Pull request by akohler**
…a_search modules.

This adds an ArticlesPlus tab to the homepage search form.

commit 9f471f6b978478dfe8b0a89eccd5c8a4beb6b469
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 14:15:50 2016 -0900

    DRUP-815: Add description text and placeholder image for ArticlesPlus tab

commit 8290fe4e10ad40b0c40039fd1bc7549ede6d066f
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 13:41:09 2016 -0900

    DRUP-817: Fix peer review limit to work correctly

commit a78fe41cc6a1e49ca6496d7ad5886b8bfc6e0c44
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 12:39:48 2016 -0900

    DRUP-813: Increment version to 7.x-1.5-alpha1

commit f669849589a0236bdd419621110ce2fbbaf28a18
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 12:36:30 2016 -0900

    DRUP-815: Add search handler for ArticlesPlus

commit acf3e177ee63d308e5c53f79be2e6fbcf2b72274
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 12:28:59 2016 -0900

    DRUP-815: Add basic mapping of ArticlesPlus search form to values for search/log handlers

commit 0c134ced5c28f2304e38010fa00582bd1887961f
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 10:46:26 2016 -0900

    DRUP-817: Add checkbox for peer review limit

commit 9ad20c1ef64ead96b45c0f7241edc44c9ac60e03
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 10:29:56 2016 -0900

    DRUP-816: Add Advanced Search link, though not opening in new tab

commit aa405795f1bac20c7875a780e275c76d1f2e9af2
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 10:22:54 2016 -0900

    DRUP-815: Add Search button to ArticlesPlus tab

commit b27e097b9c042f7f2daf2969db758d57438110ff
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Wed Feb 24 10:17:17 2016 -0900

    DRUP-815: Add ArticlesPlus to tabbed search, with basic search box

### CODE REVIEW

**Comment by akohler** [comment]
@z3cka Can you please review and if all is OK, merge & deploy to www-test?

I'll update Jira issues (many of them...) once this is on www-test for Sharon to review.  Thanks --Andy

File: www/sites/all/modules/custom/ucla_search/includes/ucla_search_handlers.inc [hunk 1]
```diff
@@ -75,6 +78,17 @@ function serial_solutions_360($search_info) {
   drupal_goto($path = $target_url);
 }
 
+function summon_articlesplus($search_info) {
+  $target_url = 'http://ucla.summon.serialssolutions.com/#!/search';
+  $extra_params = '?ho=t&l=en';
+  if ( isset($search_info['search_limit']) ) {
+    $extra_params .= '&fvf=' . $search_info['search_limit']; // debugging
```
Comment (z3cka) [inline www/sites/all/modules/custom/ucla_search/includes/ucla_search_handlers.inc]: What's this `// debugging` doing?

File: www/sites/all/modules/custom/ucla_tab_search/ucla_tab_search.info [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name = UCLA Tab Search
 description = Custom tab forms for searching on the UCLA Library homepage.
 core = 7.x
-version = "7.x-1.4"
+version = "7.x-1.5-alpha1"
```
Comment (z3cka) [inline www/sites/all/modules/custom/ucla_tab_search/ucla_tab_search.info]: typically when pull requesting to master _(`www-test`)_ we use `-betaX`, `-alphaX` is for when sharing partial work with the team in dev before an actual PR to master for QA on `www-test`.

File: www/sites/all/modules/custom/ucla_search/includes/ucla_search_handlers.inc (hunk 1 above)
Comment (akohler) [inline www/sites/all/modules/custom/ucla_search/includes/ucla_search_handlers.inc]: // debugging is left over, I'll take it out.

File: www/sites/all/modules/custom/ucla_tab_search/ucla_tab_search.info (hunk 2 above)
Comment (akohler) [inline www/sites/all/modules/custom/ucla_tab_search/ucla_tab_search.info]: @z3cka wrote:
> -alphaX is for when sharing partial work with the team in dev before an actual PR to master for QA on www-test

Ah, thanks, I was unsure.  Is this (plus the errant debugging comment) something I should fix and do a new PR for?

Comment (z3cka) [inline www/sites/all/modules/custom/ucla_tab_search/ucla_tab_search.info]: no, we can up the version during the PR to staging/prod

**Comment by z3cka** [comment]
deploying now

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

**Pull request merged**